docs/API/v3/registration-base-url-resource.md

```diff
@@ -0,0 +1,301 @@
+fetch metadata about individual package versions. The `items` array is sometimes excluded from the page
+object as an optimization. If the number of versions of a single package IDs is very large, then the registration index
+document will be massive and wasteful to process for a client that only cares about a specific version or small range
+of versions.
+
+Note that is the `items` property is present, the `@id` property need not be used, since all of the page data is
+already inlined in the `items` property.
+
+Each item in the page object's `items` array is a JSON object representing a registration leaf and it's associated
+metadata.
+
+#### Registration leaf object in a page
+
+The registration leaf object found in a registration page has the following properties:
+
+Name           | Type   | Required | Notes
+-------------- | ------ | -------- | -----
+@id            | string | yes      | The URL to the registration leaf
+catalogEntry   | object | yes      | The catalog entry containing the package metadata
+packageContent | string | yes      | The URL to the package content (.nupkg)
+
+Each registration leaf object represents data associated with a single package version.
+
+#### Catalog entry
+
+The `catalogEntry` property in the registration leaf object has the following properties:
+
+Name                     | Type                       | Required | Notes
+------------------------ | -------------------------- | -------- | -----
+@id                      | string                     | yes      | The URL to document used to produce this object
+authors                  | string or array of strings | no       | 
+dependencyGroups         | array of objects           | yes      | The URL to the package content (.nupkg)
+description              | string                     | no       | 
+iconUrl                  | string                     | no       | 
+id                       | string                     | yes      | The ID of the package
+licenseUrl               | string                     | no       | 
+listed                   | boolean                    | no       | Should be considered as listed if absent
+minClientVersion         | string                     | no       | 
+projectUrl               | string                     | no       | 
+published                | string                     | no       | A string containing a ISO 8601 timestamp of when the package was published
+requireLicenseAcceptance | boolean                    | no       | 
+summary                  | string                     | no       | 
+tags                     | string or array of string  | no       | 
+title                    | string                     | no       | 
+version                  | string                     | yes      | The version of the package
+
+The `dependencyGroups` property is an array of objects representing the dependencies of the package, grouped by target
+framework.
```

Comment:
Perhaps elaborate what this looks like for a package without dependencies.

Reply:
Fixed.